cmd/readiness/main.go

```diff
@@ -138,12 +136,18 @@ func findCurrentStep(processStatuses map[string]health.MmsDirectorStatus) *healt
 	return lastStartedStep
 }
 
-func isDeadlocked(status *health.StepStatus) bool {
+// isWaitStep return true is the Agent is currently waiting for something.
```

Comment:
Can we add a bit more context what a `waitstep` is or why it matters here to return "true"?

Reply:
Done